.circleci/config.yml

```diff
@@ -113,7 +113,7 @@ aliases:
     requires:
       - lint
       - typecheck
-      - unit_tests_node18
+      # - unit_tests_node18
```

Comment:
this was just to get integration tests to run sooner - this should be reverted once things are figured out